packages/web-components/fast-components/src/tabs/tab/tab.styles.ts

```diff
@@ -24,25 +34,45 @@ export const TabStyles = css`
         font-weight: 400;
         line-height: 18px;
         height: calc(${heightNumber} * 1px);
-        padding: 0 calc((6 + (var(--design-unit) * 2 * var(--density))) * 1px);
-        color: var(--neutral-foreground-rest);
+        padding: calc(var(--design-unit) * 5px) calc(var(--design-unit) * 4px);
+        color: var(--neutral-foreground-hint);
+        fill: var(--neutral-foreground-hint);
         border-radius: calc(var(--corner-radius) * 1px);
         border: calc(var(--outline-width) * 1px) solid transparent;
         align-items: center;
         justify-content: center;
         grid-row: 1;
     }
 
-    :host([aria-selected="true"]) {
-        z-index: 2;
-    }
-
     :host(:hover) {
         color: var(--neutral-foreground-hover);
+        fill: var(--neutral-foreground-hover);
     }
 
     :host(:active) {
         color: var(--neutral-foreground-active);
+        fill: var(--neutral-foreground-active);
+    }
+
+    :host([aria-selected="true"]) {
+        z-index: 2;
```

Comment:
Any way to do this without z-index. Seems like it shouldn't be used for a flat component like this. Something to do with the indicator moving between? Can it stay fixed if we need it? Also, can it be defined only once on the base style, not each state?

Reply:
appears we can, that was in the original file and not sure why it was that way orginally